OrbitClientGgp/ClientGgp.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ClientGgp.h"
+
+#include "capture_data.pb.h"
+
+#include "Capture.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/Result.h"
+
```

Comment:
We should try to explicitly include the headers needed here.

- `std::numeric_limits` needs `<limits>`.
- `std::make_unique` needs `<memory>`. 
- `int32_t` needs `<cstdint>`.
- `std::map` needs `<map>`.
- `std::string` needs `<string>`.

Reply:
Done. I was trying to avoid including headers that are already included by including others; but maybe it makes sense for these, so I have included them.

Reply:
Yeah, that's always a thing for debate what headers to include. The Google C++ style guide is pretty clear here though: We should "include all headers defining symbols we rely upon".

Thanks!